scripts/variantstore/wdl/extract/hail_gvs_import.py

```diff
@@ -30,7 +30,8 @@ def import_gvs(argsfn, vds_path, references_path, temp_path):
     rg38.add_sequence(f'{references_path}/Homo_sapiens_assembly38.fasta.gz',
                       f'{references_path}/Homo_sapiens_assembly38.fasta.fai')
 
-
+    ## Note that a full description of the import_gvs function written by Hail for this process can be found here: hail/python/hail/methods/impex.py
```

Comment:
Here's a [reference to the file](https://github.com/tpoterba/hail/blob/import-gvs/hail/python/hail/methods/impex.py#L3022) as it currently exists only in Tim's fork and not in the main Hail repo.

Reply:
so I should remove this? I dont understand...